tests/util.cpp

```diff
@@ -27,5 +27,14 @@ main()
   TEST_ASSERT(btllib::startswith(actg_join, "A/C"));
   TEST_ASSERT(btllib::endswith(actg_join, "/T/G"));
 
+  std::string qual = "123456789";
```

Comment:
Might be good to have another test with a more representative phred string? (ie. with non-numerical characters)